contracts/nft/NFTFacet.sol

```diff
@@ -56,4 +56,23 @@ contract NFTFacet is RolesMods {
         // Give the caller authorization to protocol
         RolesLib.grantRole(AUTHORIZED, msg.sender);
     }
+
+    /**
+     * @notice It returns the claimable interest % for a user's staked NFTs.
+     * @param nftOwner The address of the NFT owner.
+     */
+    function getClaimableInterestPercent(address nftOwner)
```

Comment:
lets create a new facet so that we don't have to redeploy existing logic on this facet

Reply:
done